libs/api/src/main/java/de/cyface/api/MeasurementRetrievalImpl.java

```diff
@@ -1,115 +0,0 @@
-/*
- * Copyright 2021-2022 Cyface GmbH
- *
- * This file is part of the Cyface Data Collector.
- *
- * The Cyface Data Collector is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * The Cyface Data Collector is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
- */
-package de.cyface.api;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import org.apache.commons.lang3.Validate;
-
-import de.cyface.model.MeasurementIdentifier;
-import de.cyface.model.MetaData;
-import de.cyface.model.Modality;
-import de.cyface.model.Point3DImpl;
-import de.cyface.model.RawRecord;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-
-/**
- * This class defines methods required by all {@link MeasurementRetrievalStrategy} implementations.
- *
- * @author Armin Schnabel
- * @since 6.0.0
- */
-public abstract class MeasurementRetrievalImpl implements MeasurementRetrievalStrategy {
```

Comment:
This class was never used and the method implementations are already `default` implementations in the interface.